Makefile

```diff
@@ -15,9 +15,13 @@ bin_out := $(OUT_DIR)/tracee
 .PHONY: build
 build: $(bin_out)
 
-$(bin_out): $(bpf_out) $(go_src) | $(OUT_DIR)
+$(bin_out)_slim: $(bpf_out) $(go_src) | $(OUT_DIR)
 	GOOS=linux GOARCH=$(ARCH:x86_64=amd64) go build -v -o $(bin_out)
 
+$(bin_out): $(bpf_out) $(go_src) | $(OUT_DIR)
+	GOOS=linux GOARCH=$(ARCH:x86_64=amd64) go build -v -o $(bin_out) \
```

Comment:
I don't think we should embed the binary in the code, as this is architecture specific.
See my other comment in the makefile